CMakeLists.txt

```diff
@@ -30,63 +30,35 @@
 
 #]=======================================================================]
 
-project(OpenVDB_AX)
 cmake_minimum_required(VERSION 3.3)
+project(OpenVDB_AX)
 
 include(CMakeDependentOption)
 
 option(OPENVDB_BUILD_AX "Build the OpenVDB AX library" ON)
 option(OPENVDB_BUILD_AX_BINARIES "Build the OpenVDB AX command line binary" ON)
-option(OPENVDB_BUILD_AX_PYTHON_MODULE "Build the OpenVDB AX Python module" ON)
+option(OPENVDB_BUILD_AX_PYTHON_MODULE "Build the OpenVDB AX Python module" OFF)
 option(OPENVDB_BUILD_AX_HOUDINI_PLUGIN "Build the OpenVDB AX Houdini plugin" OFF)
 option(OPENVDB_BUILD_AX_UNITTESTS "Build the OpenVDB AX unit tests" OFF)
 option(OPENVDB_BUILD_AX_DOCS "Build the OpenVDB AX documentation" OFF)
 option(USE_CCACHE "Build using Ccache if found on the path" ON)
 option(USE_HOUDINI "Build the library against a Houdini installation." OFF)
-option(USE_SYSTEM_LIBRARY_PATHS "Build with system library paths" ON)
+option(DISABLE_CMAKE_SEARCH_PATHS [=[
+Disable CMakes default system search paths when locating dependencies. When enabled, CMake will fall back to
+its default system search routine if it cannot find a dependency with the provided settings. When disabled, only
+paths provided through the Xxx_ROOT, supported XXX_INCLUDEDIR/XXX_LIBRARYDIR variables or the SYSTEM_LIBRARY_PATHS
+list will be searched.]=] OFF)
 
 ###### Deprecated options
 
-option(OPENVDB_AX_BUILD_AX "Build the OpenVDB AX compiler" OFF)
-option(OPENVDB_AX_BUILD_PYTHON_BINDINGS "Build the OpenVDB AX Python bindings" OFF)
-option(OPENVDB_AX_BUILD_GRAMMAR "Rebuild the OpenVDB AX grammar. CMake 3.4 with flex and bison." OFF)
-option(OPENVDB_AX_BUILD_UNITTESTS "Build the OpenVDB AX unit tests" OFF)
-option(OPENVDB_AX_BUILD_DOCS "Build the OpenVDB AX documentation" OFF)
-option(OPENVDB_AX_BUILD_HOUDINI_SOP "Build the OpenVDB AX Houdini SOP" OFF)
-option(OPENVDB_AX_ENABLE_RPATH "Build with RPATH information" OFF)
-option(OPENVDB_AX_CXX_STRICT "Enable or disable pre-defined compiler warnings" OFF)
-
-if(OPENVDB_AX_BUILD_AX)
```

Comment:
Were these only just deprecated in 0.1.0? Maybe should re-instate until 0.2.0?

Reply:
done